src/server.js

```diff
@@ -1,18 +1,27 @@
 import express from 'express';
-import { userRouter, articlesRouter, otherRouter } from './routes';
+import passport from 'passport';
+import {
+  userRouter,
+  articlesRouter,
+  otherRouter,
+  socialAuthRouter
+} from './routes';
 import registerMiddlewares from './middlewares';
+import './services/passport';
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 registerMiddlewares(app);
 
+app.use(passport.initialize());
```

Comment:
You've done a great job implementing this feature. However, I feel there's no need to clutter the `server.js` file with different middleware registrations. We have a `middlwares/index.js` file which serves as a single source of truth for all middlewares the app uses. Here on out, please, endeavor to register new middlewares in the `middlewares/index.js` file.  Cheers 👍

Reply:
Thanks for the feedback, Luqman. 👍